tests/integration/plugins.js

```diff
@@ -416,4 +421,29 @@ describe('Integration: plugins', () => {
       `)
     })
   })
+
+  describe('jss-nested', () => {
+    let sheet
+
+    beforeEach(() => {
+      jss.use(jssNested())
+
+      sheet = jss.createStyleSheet({}, {
+        link: true,
+      }).attach()
+
+      sheet.addRule('b', {color: 'green'})
+      sheet.addRule('a', {
+        '&:hover': {
+          '& $b': {
+            color: 'red',
+          },
+        },
+      })
+    })
+
+    it('should save the added nested rules order', () => {
+      expect(getCssFromSheet(sheet)).to.be(removeWhitespace(sheet.toString()))
```

Comment:
we only use DOM based tests in functional tests, is DOM really needed here or sheet.toString() + comparing with result would be enough?

Reply:
yeah, I think we can replace expect with the exact result